src/SportsHub.Web/Controllers/TeamsController.cs

```diff
@@ -87,5 +94,42 @@ public async Task<IActionResult> EditTeam([FromForm] EditTeamModel editTeamModel
 
             return Ok();
         }
+
+        [HttpGet("{location}/{categoryName}/{subcategoryName}")]
+        /*[Authorize(Policies.Admin)]*/
+        [AllowAnonymous]
```

Comment:
Same here.
Sorry, but why do you need this mock? I'm asking because you can easily get an access token and get authorized by it. Please, ask @foxerest or @markoparfeniuk how to get an access token for our API.